tdrs-frontend/nginx/nginx.conf

```diff
@@ -12,8 +12,4 @@ server {
     add_header Content-Security-Policy "default-src 'self'; *.acf.hhs.gov; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self'; connect-src 'self'; *.acf.hhs.gov manifest-src 'self'; object-src 'none'; frame-ancestors 'none'; form-action 'none';";
     add_header Access-Control-Allow-Origin "https://tanfdata.acf.hhs.gov";
 
-     location / {
-         proxy_pass http://$host;
-         proxy_set_header Host $host;
-     }
```

Comment:
This part is what ultimately broke the scans. It seems nginx was attempting to proxy traffic to somewhere that does not exist.

Reply:
I deployed this to dev raft and it still works. The `add_header Access-Control-Allow-Origin "https://tanfdata.acf.hhs.gov";` makes it seem like this nginx conf is for production, but on closer inspection that does not seem to be the case. The buildpack.nginx.conf seems to be used for that. This line is likely not needed here, thus this change should not have an effect on production.

Still we should test this commit on prod or be ready to revert it if it does break something.